src/main/java/ca/tirelesstraveler/fancywarpmenu/gui/GuiFancyWarp.java

```diff
@@ -171,7 +172,13 @@ public void onWarpFail(String failMessageKey) {
     protected void actionPerformed(GuiButton button) {
         // Block repeat clicks if the last warp failed
         if (button instanceof GuiWarpButton && Minecraft.getSystemTime() > warpFailCoolDownExpiryTime) {
-            sendChatMessage(((GuiWarpButton)button).getWarpCommand());
+            String warpCommand = ((GuiWarpButton) button).getWarpCommand();
+            try {
+                mc.ingameGUI.getChatGUI().addToSentMessages(warpCommand);
+                mc.thePlayer.sendQueue.addToSendQueue(new C01PacketChatMessage(warpCommand));
```

Comment:
Could we make the mod send commands as “/fancywarp hub” instead of /warp? That way other mods can still intercept without packet listening. The hook can remove the “fancy” before sending.

Reply:
You want to add support for other mods to mixin into the addCommand method to detect or even block if fancy warp menu sends a command? I'm not sure why we would want to do this.
If we send the command as fancywarp other mods would have to work with additional edge cases. Instead atm, other mods can't access the exact command. Why should they?

Reply:
I wanted them to be able to see the command by mixing into `EntityPlayerSP#sendChatMessage`, which some mods mix in to. I don’t know any that look for warp commands specifically but wanted to keep the option open. You’re right about the edge cases, in case their mixins are inserted before ours. If a mod turns out to need access, we can see if something like `/warp hub fancy` is a better solution.